src/app/auth/features/user-manager-detail/user-manager-detail.component.html

```diff
@@ -0,0 +1,96 @@
+@if (user(); as currentUser) {
+  <div class="container mx-auto p-6">
+    <div class="clearfix mb-6">
+      <button
+        mat-icon-button
+        (click)="goBack()"
+        aria-label="Go back to user list"
+        class="text-primary float-left mt-1 mr-4"
```

Comment:
```suggestion
        class="text-primary md:float-left mt-1 mr-4"
```
For a better look on small screens